code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -884,3 +884,68 @@
 
 /datum/reagent/toxin/mimesbane/on_mob_delete(mob/living/L)
 	L.remove_trait(TRAIT_EMOTEMUTE, id)
+
+/datum/reagent/toxin/bonehurtingjuice //oof ouch
+	name = "Bone Hurting Juice"
+	id = "bonehurtingjuice"
+	description = "A strange substance that looks a lot like water. Drinking it is oddly tempting. Oof ouch."
+	color = "#AAAAAA77"
+	toxpwr = 0
+	taste_description = "bone hurting"
+	overdose_threshold = 50
+	metabolization_rate = 2.0
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_add(mob/living/carbon/M)
+	M.say("oof ouch my bones")
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(8.0, 0)
+	if(prob(20))
+		switch(pick(1, 2, 3))
+			if(1)
+				var/list/possibleSays = list("oof", "ouch", "my bones", "oof ouch", "oof ouch my bones")
+				M.say(pick(possibleSays))
+			if(2)
+				var/list/possibleMes = list("oofs softly", "looks like their bones hurt", "grimaces, as though their bones hurt")
+				M.say("*custom " + pick(possibleMes))
+			if(3)
+				to_chat(M, "<span class='warning'>Your bones hurt!</span>")
+	. = ..()
+
+/datum/reagent/toxin/bonehurtingjuice/overdose_process(mob/living/M)
+	var/obj/item/bodypart/l_arm = M.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/r_arm = M.get_bodypart(BODY_ZONE_R_ARM)
+	var/obj/item/bodypart/l_leg = M.get_bodypart(BODY_ZONE_L_LEG)
+	var/obj/item/bodypart/r_leg = M.get_bodypart(BODY_ZONE_R_LEG)
```

Comment:
You should only need one var

Reply:
I'm not really sure what you mean by this, but I just dropped all the vars entirely in favor of direct references.  Does it error if you attempt to damage a missing limb though?